src/System.Threading/tests/EventWaitHandleTests.cs

```diff
@@ -224,5 +227,10 @@ private static int PingPong_OtherProcess(string modeName, string inboundName, st
             return SuccessExitCode;
         }
 
+        public static TheoryData<string> GetValidNames => new TheoryData<string>()
+        {
+            { Guid.NewGuid().ToString("N") },
+            { Guid.NewGuid().ToString("N") + new string('a', 1000) }
```

Comment:
Does this case need to be disabled for .NET Framework?

Reply:
Yeah I will disable it. should I re-add the test for ```Invalid Name``` for just .NET Framework ?